src/item.cpp

```diff
@@ -5581,13 +5698,42 @@ layer_level item::get_layer() const
     }
 }
 
-int item::get_coverage() const
+int item::get_coverage( const bodypart_id &bodypart ) const
 {
     const islot_armor *t = find_armor_data();
     if( t == nullptr ) {
         return 0;
     }
-    return t->coverage;
+
+    int avg_coverage = 0;
+    int avg_ctr = 0;
+    if( bodypart == bodypart_id( bodypart_str_id( "num_bp" ) ) ) {
```

Comment:
I think it would be better to have a separate `get_average_coverage` function than overload the purpose of this one.